keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/ClusterDeployStrategy.kt

```diff
@@ -1,10 +1,12 @@
 package com.netflix.spinnaker.keel.api
 
 import com.netflix.spinnaker.keel.api.DeployHealth.AUTO
+import com.netflix.spinnaker.keel.api.schema.Discriminator
 import java.time.Duration
 import java.time.Duration.ZERO
 
-sealed class ClusterDeployStrategy {
+abstract class ClusterDeployStrategy {
+  @Discriminator abstract val strategy: String
```

Comment:
I had to make this change in order that the schema generator outputs something where a validator can tell the difference between `RedBlack` and `Highlander`. It makes no difference to the YAML format.